Dockerfile

```diff
@@ -1,10 +1,11 @@
 FROM frolvlad/alpine-glibc
 
 RUN apk add --update ca-certificates \
-    && rm -rf /var/cache/apk/*
+    && rm -rf /var/cache/apk/*\
+    && mkdir /embedded
 
 ADD ./mayu-infopusher /mayu-infopusher
-ADD ./embedded/ /embedded
+ADD ./embedded/* /embedded/
```

Comment:
Let's be specific here, we've had issues in the past with wildcards adding extraneous files.